integrations/pjrt/README.md

```diff
@@ -16,9 +16,8 @@ most powerful).
 ```
 pip install -r requirements.txt
 
-# Assume that you have the Jax repo checked out at JAX_REPO from
-# https://github.com/google/jax (must be paired with nightly jaxlib).
-pip install -e $JAX_REPO
+# a higher version of jax is highly recommended, e.g. 0.4.37
+pip install jax==0.4.37
```

Comment:
Here I want to recommend users/devs to install the latest version, manually. But seems we cannot update this for every version of jax.

Reply:
Ideally we would specify a minimum version or a version range. Setting to an exact version that is tested by the CI here is a fine place to be right now.

Reply:
Ahh good idea. I changed it to 0.4.36 since it's the version used in CI.